src/source/benches/json_parser.rs

```diff
@@ -58,62 +58,71 @@ fn get_descs() -> Vec<SourceColumnDesc> {
             data_type: DataType::Int32,
             column_id: ColumnId::from(0),
             skip_parse: false,
+            is_timestamp: false,
```

Comment:
I'm not sure what you means, do you means we can have a interface to create SourceColumnDesc, like 
`SourceColumnDesc::new(data_type,column_id,skip_parse)` so that we don't need to specify the is_timstamp field for the mqs that have no timestamp with messages?

Reply:
But that may cause parser have some 'hacky' code like: 
```
if column.name == '_rw_kafka_timestamp' || 
   column.name == '_rw_**_timestamp' {
    ...
}
```
is that accpectable?